templates/terraform/examples/service_directory_endpoint_basic.tf.erb

```diff
@@ -0,0 +1,25 @@
+resource "google_service_directory_namespace" "<%= ctx[:primary_resource_id] %>" {
+  provider     = google-beta
+  namespace_id = "<%= ctx[:vars]["namespace_id"] %>"
+  location     = "us-central1"
+}
+
+resource "google_service_directory_service" "<%= ctx[:primary_resource_id] %>" {
+  provider   = google-beta
+  service_id = "<%= ctx[:vars]["service_id"] %>"
+  namespace  = google_service_directory_namespace.<%= ctx[:primary_resource_id] %>.id
+}
+
+resource "google_service_directory_endpoint" "<%= ctx[:primary_resource_id] %>" {
```

Comment:
it would be helpful for all of these to also have a test that sets only required fields- that way, we can verify that optional fields don't end up with api-side defaults that we need to handle by making the  field computed.

Reply:
Removed optional fields from basic custom test and left optionals only for the updates.